composeApp/src/commonMain/kotlin/io/github/drp08/studypal/screens/CalendarScreen.kt

```diff
@@ -1,12 +1,114 @@
 package io.github.drp08.studypal.screens
 
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import cafe.adriel.voyager.core.screen.Screen
+import kotlinx.datetime.*
 
 object CalendarScreen : Screen {
+
+    data class Event(val startTime: LocalTime, val endTime: LocalTime, val title: String)
+
     @Composable
     override fun Content() {
-        Text(text = "Calendar Screen - Testing for deployment")
+        val startTime = LocalTime(8, 0) // Start time
+        val endTime = LocalTime(18, 0) // End time
+        val events = listOf(
+            Event(LocalTime(9, 0), LocalTime(10, 0), "Meeting 1"),
+            Event(LocalTime(11, 0), LocalTime(12, 30), "Lunch"),
+            Event(LocalTime(13, 0), LocalTime(15, 0), "Meeting 2"),
+            Event(LocalTime(15, 30), LocalTime(17, 0), "Workshop")
+        )
```

Comment:
This should be in a `ViewModel` because we will later fetch this from the source.